lib/core/enums/collect_group_type.dart

```diff
@@ -45,14 +46,29 @@ enum CollectGroupType {
     }
   }
 
+  static ColorCombo getColorComboByType(CollectGroupType type) {
+    switch (type) {
+      case CollectGroupType.church:
+        return ColorCombo.primary;
+      case CollectGroupType.charities:
+        return ColorCombo.tertiary;
+      case CollectGroupType.campaign:
+        return ColorCombo.highlight;
+      case CollectGroupType.artists:
+        return ColorCombo.secondary;
+      default:
+    }
+    return ColorCombo.secondary;
+  }
+
   static IconData getIconByType(CollectGroupType type) {
     switch (type) {
       case CollectGroupType.church:
         return FontAwesomeIcons.placeOfWorship;
       case CollectGroupType.charities:
         return FontAwesomeIcons.solidHeart;
       case CollectGroupType.campaign:
-        return FontAwesomeIcons.handHoldingHeart;
+        return FontAwesomeIcons.solidFlag;
```

Comment:
Maybe these icons should be copied to Family/US section of the app. Otherwise we are changing it for the EU as well I guess

Reply:
True but I do feel like the EU icons are very busy and ugly. Especially the handholdingheart. Maybe we can get Camilla's amen for this XD

Reply:
oh i did separate it, but yea